packages/core/src/CSL/cslToCore/certificate.ts

```diff
@@ -34,52 +35,54 @@ const stakeDelegation = (certificate: CSL.StakeDelegation): StakeDelegationCerti
   stakeKeyHash: Ed25519KeyHash(Buffer.from(certificate.stake_credential().to_keyhash()!.to_bytes()).toString('hex'))
 });
 
-const createCardanoRelays = (relays: CSL.Relays): Relay[] => {
-  const result: Relay[] = [];
-  for (let i = 0; i < relays.len(); i++) {
-    const relay = relays.get(i);
-    const relayByAddress = relay.as_single_host_addr();
-    const relayByName = relay.as_single_host_name();
-    const relayByNameMultihost = relay.as_multi_host_name();
+const createCardanoRelays = (relays: CSL.Relays): Relay[] =>
+  usingAutoFree((scope) => {
+    const result: Relay[] = [];
+    for (let i = 0; i < relays.len(); i++) {
+      const relay = scope.manage(relays.get(i));
+      const relayByAddress = relay.as_single_host_addr();
+      const relayByName = relay.as_single_host_name();
+      const relayByNameMultihost = relay.as_multi_host_name();
```

Comment:
Don't these need to be managed?

Reply:
214d62f11644c689a1e38c3f1811f88e87acf8ac